tools/KoreBuild.Console/Commands/DockerFiles/winservercore.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/aspnet:4.6.2
+
+ADD https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe /net461dev.exe
```

Comment:
You should be able to download, install, and remove in a single command (which is more efficient) like so:

https://github.com/Microsoft/aspnet-docker/blob/master/4.6.2/Dockerfile#L12-L16

Reply:
This is actually somewhat complicated since `NDP461-DevPack-KB3105179-ENU.exe /Passive /NoRestart` immediately exits with error code 0, and the install runs in the background.  So, the delete will fail unless you wait a bit after starting the install.  In your original Dockerfile, I think there's a small delay between the two run commands which usually causes the delete to succeed, but it's not guaranteed.  And technically, I don't think it's guaranteed the install will be complete by the time `build.cmd` tries to use it.  The following should work, but there's no way to know the correct length of time to sleep:

```
RUN Invoke-WebRequest https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe \
      -OutFile C:\NDP461-DevPack-KB3105179-ENU.exe; \
    C:\NDP461-DevPack-KB3105179-ENU.exe /Passive /NoRestart; \
    Start-Sleep -s 10; \
    Remove-Item C:\NDP461-DevPack-KB3105179-ENU.exe -Force;
```

The most reliable way would be to block until we know the install is complete, perhaps using the existence of a file or registry key.